### PR TITLE
Create store

### PR DESCRIPTION
**Pull request by omorilewa**
#### What does this PR do?
Allows the seller to create a store

#### Description of Task to be completed?
Rework name on navbar
Account for page refresh with id and crypto js
Allow seller to create a store
Notification after store creation
Minor cleanup

#### How should this be manually tested?
Signup a new user
it should redirect to the create store page
Fill relevant details
Click submit
Ensure your env var is set
You should get a notification that the store was created
Error message on Login/Signup

#### Any background context you want to provide?
I put a next button because creating a store would be a two step process, the next step would be adding account details and any other necessary info

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/154254482

#### Screenshots (if appropriate)
Screenshot 2019-06-19 at 12 32 17 PM

#### Questions:
Taking qustions

### CODE REVIEW

File: src/apollo-client/resolvers.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import gql from 'graphql-tag';
 
 export const typeDefs = gql`
   extend type Query {
-    username: String
+    user: UserType
```
Comment (omorilewa) [inline src/apollo-client/resolvers.js]: Leaving this here for when next I need it.

File: src/app/App.jsx [hunk 2]
```diff
@@ -24,6 +26,7 @@ function App({ classes }) {
           <Route path="/" exact component={LandingPage} />
           <Route path="/login" exact component={AuthPage} />
           <Route path="/createproduct" exact component={CreateProduct} />
+          <Route path="/createstore" exact component={CreateStore} />
```
Comment (fisayomi) [inline src/app/App.jsx]: Can we make this route protected?

Comment (omorilewa) [reply src/app/App.jsx]: What error do you get on login? Login looks good for me locally.

**Comment by omorilewa** [comment]
I updated the PR to prevent Login from redirecting to the create store page. It would redirect to the store page when the page exists.

File: src/app/App.jsx (hunk 2 above)
Comment (fisayomi) [reply src/app/App.jsx]: > What error do you get on login? Login looks good for me locally.

   Error: Network error: Response not successful: Received status code 400

File: src/components/Store/create-store.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+const CreateStore = () => {
+  const userId = decrypt(localStorage.getItem('user') || '');
+  const initialValues = {
+    name: '',
+    phonePrimary: '',
+    phoneSecondary: ''
+  };
+  return (
+    <Mutation mutation={CREATE_STORE}>
+      {(createStore, { loading }) => (
```
Comment (fisayomi) [inline src/components/Store/create-store.js]: We need to also get errors from here

File: src/apollo-client/resolvers.js (hunk 1 above)
Comment (oreoluwade) [reply src/apollo-client/resolvers.js]: Yes ma'am

File: src/components/protected/protected.jsx [hunk 4]
```diff
@@ -1,23 +1,27 @@
...
-  isAuthenticated: bool
-};
+export default Protected;
 
-const mapStateToProps = state => {
-  return {
-    isAuthenticated: state.user.isAuthenticated
-  };
-};
+// Protected.propTypes = {
```
Comment (oreoluwade) [inline src/components/protected/protected.jsx]: Is this part commented for future editing?

File: src/components/Store/create-store.js (hunk 3 above)
Comment (omorilewa) [reply src/components/Store/create-store.js]: Why do I need to get the errors here?

File: src/components/protected/protected.jsx (hunk 4 above)
Comment (omorilewa) [reply src/components/protected/protected.jsx]: Lol!

**Pull request merged**